remixer_example/google-services.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
That's fine. Though I expect some documentation explaining this to a reader. What should the user do? Replace this file with his/her own? Etc